dev/io.openliberty.microprofile.reactive.streams.operators30.internal_fat_tck/publish/servers/ReactiveStreams30TCKServer/server.xml

```diff
@@ -0,0 +1,24 @@
+<!--Copyright (c) 2018, 2023 IBM Corporation and others. All rights reserved.
+    This program and the accompanying materials are made available under the 
+    terms of the Eclipse Public License 2.0 which accompanies this distribution, 
+    and is available at 
+        http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0 
+-->
+<server>
+    <featureManager>
+        <feature>servlet-6.0</feature>
```

Comment:
Could use the rest protocol and remove servlet dependency as we do for the other MP features.

I think it's less of an issue here since reactive streams doesn't interact with either servlet or JAX-RS